OrrRock/OrrRock/Models/UploadModels/SwipeableCardModel/SwipeableCardVideoModel.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SwipeableCardVideoModel.swift
+//  OrrRock
+//
+//  Created by Ruyha on 2022/11/08.
+//
+
+import Foundation
+
+enum VideoBackgroundViewBorderColor{
```

Comment:
```suggestion
enum VideoBackgroundViewBorderColor {
```

p1; 

한 칸 띄어주는게 읽기 편할 것 같아요 ㅎ